homeassistant/components/ezviz/camera.py

```diff
@@ -340,9 +356,8 @@ async def stream_source(self):
             return rtsp_stream_source
         return None
 
-    def perform_ptz(self, direction, speed):
+    def perform_ptz(self, direction, speed) -> None:
```

Comment:
Please type the whole signature when adding type annotations to a function.

Reply:
Hi @MartinHjelmare,

Done. Added type annotations for the whole signature in all functions.